libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Extensions/PushNotificationManager.swift

```diff
@@ -111,14 +111,19 @@ extension PushNotificationManager {
     }
     
     private func fetchNotificationTypesFromAPI(with client: RestClient) async throws -> [NotificationType] {
-        guard client.apiVersion.compare("v61.0").rawValue >= 0 else {
-            throw PushNotificationManagerError.notificationActionInvocationFailed("API Version must be at least v61.0")
+        guard client.apiVersion.compare("v64.0").rawValue >= 0 else {
+            throw PushNotificationManagerError.notificationActionInvocationFailed("API Version must be at least v64.0")
         }
         
-        let request = RestRequest(method: .GET, path: "connect/notifications/types", queryParams: nil)
+        let request = RestRequest(method: .GET, path: "\(client.apiVersion)/connect/notifications/types", queryParams: nil)
```

Comment:
I manually add the apiVersion here.  When  making the request the version wasn't being added to the URL as I assumed it was done dureing the prepare phase of the reqeust, but it wasn't.

Reply:
You could also add a RestRequest factory method in SFRestAPI.m. Most take apiVersion as a parameter, but allows nil to be passed. When nil is passed the apiVersion used is the default one set on the SFRestAPI instance.
See https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/dev/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m#L514

Reply:
ah, what's the prefered way you think here?

Reply:
I was thinking new rest request factory methods for notifications related stuff.
Simply call them from here.

Reply:
Cool, done.  Hopefully this change I made was what you had in mind. @wmathurin